src/AppBundle/EventListener/CheckRequirementsSubscriber.php

```diff
@@ -54,9 +50,9 @@ public static function getSubscribedEvents()
         return [
             // Exceptions are one of the events defined by the Console. See the
             // rest here: http://symfony.com/doc/current/components/console/events.html
-            ConsoleEvents::EXCEPTION => 'handleDatabaseExceptions',
-            // See: http://api.symfony.com/3.2/Symfony/Component/HttpKernel/KernelEvents.html
-            KernelEvents::EXCEPTION => 'onKernelException',
+            ConsoleEvents::EXCEPTION => 'handleConsoleException',
+            // See: http://api.symfony.com/master/Symfony/Component/HttpKernel/KernelEvents.html
```

Comment:
It was my first try, but ...

![api_current](https://cloud.githubusercontent.com/assets/73419/22683915/784ca438-ed1a-11e6-8c90-60a3c39e16d4.png)